### PR TITLE
Fix lso operator channel

### LINKED ISSUE #2324: local-storage-operator installation failed over OCP 4.5 and OCP 4.6  when used channel 4.5 | 4.6
creating temp yaml file with local-storage-operator data:
 [{'apiVersion': 'v1', 'kind': 'Namespace', 'metadata': {'name': 'local-storage'}, 'spec': {}}, {'apiVersion': 'operators.coreos.com/v1', 'kind': 'OperatorGroup', 'metadata': {'annotations': {'olm.providedAPIs': 'LocalVolume.v1.local.storage.openshift.io'}, 'name': 'local-storage', 'namespace': 'local-storage'}, 'spec': {'targetNamespaces': ['local-storage']}}, {'apiVersion': 'operators.coreos.com/v1alpha1', 'kind': 'Subscription', 'metadata': {'name': 'local-storage-operator', 'namespace': 'local-storage'}, 'spec': {'channel': '4.5', 'installPlanApproval': 'Automatic', 'name': 'local-storage-operator', 'source': 'redhat-operators', 'sourceNamespace': 'openshift-marketplace'}}]
23:27:54 - MainThread - ocs_ci.utility.templating - INFO - ['apiVersion\n--- kind\n--- metadata\n--- spec\n...\n', 'apiVersion\n--- kind\n--- metadata\n--- spec\n...\n', 'apiVersion\n--- kind\n--- metadata\n--- spec\n...\n']

LSO installation with channel 4.5 will fail as 4.5 is not yet GA so 4.5 channel is not yet opened 
as a WA we need to use n-1 as channel for LSO deployment

`Current Opened Channel`
Screenshot (56)

https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/deployment.py#L675-L677

### PR DESCRIPTION
**Pull request by prsurve**
Fixes #2324 

- Fix LSO operator channel

- Fix wrong sc name

### CODE REVIEW

**Comment by prsurve** [comment]
Verification for LSO channel :- http://pastebin.test.redhat.com/882113

**Comment by ebenahar** [comment]
@prsurve verified?

**Comment by prsurve** [comment]
Verification for OCS deployment http://pastebin.test.redhat.com/884040

File: ocs_ci/utility/localstorage.py [hunk 1]
```diff
@@ -171,6 +171,6 @@ def get_lso_channel():
     """
     channel = get_ocp_version()
     # Workaround for https://github.com/red-hat-storage/ocs-ci/issues/2324
-    if channel == '4.5':
+    if channel == '4.5' or '4.6':
```
Comment (petr-balogh) [inline ocs_ci/utility/localstorage.py:174]: I think the 4.5 is out. So maybe we can start removing those workarounds for unreleased 4.5? @clacroix12 can you maybe take care of it? I've opened issue some time back:  https://github.com/red-hat-storage/ocs-ci/issues/2390 . Thanks

**Pull request merged**